sources/views/contacts.js

```diff
@@ -25,21 +25,19 @@ export default class Contacts extends JetView {
 								);
 							},
 							css: "webix_shadow_medium app_start",
-							click: function (id) {
-								this.$scope.setParam("id", id, true);
+							click: (id) => {
+								this.setParam("id", id, true);
 							},
 							onClick: {
 								removeBtn: function (ev, id) {
 									contacts.remove(id);
 
-									const contactsList = this.$scope;
 									let firstId = this.getFirstId();
 
 									if (firstId) {
-										this.select(firstId);
-										contactsList.setParam("id", firstId, true);
+										contacts.callEvent("onStoreUpdated", [firstId]);
```

Comment:
you do not need to call webix events manually. they are called by webix code. you only need to use events and add event listeners. everything between `contacts.remove(id);` and `return false` is not needed